packages/l10n/data/time-zones/core.json

```
@@ -0,0 +1,285 @@
+  "America/Vancouver": "Pacific Time - Vancouver",
+  "America/Dawson": "Yukon Time - Dawson",
+  "America/Whitehorse": "Yukon Time - Whitehorse",
+  "Pacific/Easter": "Easter Island Time",
+  "America/Belize": "Central Standard Time - Belize",
+  "America/Costa_Rica": "Central Standard Time - Costa Rica",
+  "America/El_Salvador": "Central Standard Time - El Salvador",
+  "America/Guatemala": "Central Standard Time - Guatemala",
+  "America/Managua": "Central Standard Time - Managua",
+  "America/Regina": "Central Standard Time - Regina",
+  "America/Tegucigalpa": "Central Standard Time - Tegucigalpa",
+  "Pacific/Galapagos": "Galapagos Time",
+  "America/Boise": "Mountain Time - Boise",
+  "America/Cambridge_Bay": "Mountain Time - Cambridge Bay",
+  "America/Chihuahua": "Mountain Time - Chihuahua",
+  "America/Denver": "Mountain Time - Denver",
+  "America/Edmonton": "Mountain Time - Edmonton",
+  "America/Inuvik": "Mountain Time - Inuvik",
+  "America/Mazatlan": "Mountain Time - Mazatlan",
+  "America/Ojinaga": "Mountain Time - Ojinaga",
+  "America/Yellowknife": "Mountain Time - Yellowknife",
+  "America/Eirunepe": "Acre Standard Time - Eirunepe",
+  "America/Rio_Branco": "Acre Standard Time - Rio Branco",
+  "America/Bahia_Banderas": "Central Time - Bahia Banderas",
+  "America/North_Dakota/Beulah": "Central Time - Beulah",
+  "America/Chicago": "Central Time - Chicago",
+  "America/Matamoros": "Central Time - Matamoros",
+  "America/Menominee": "Central Time - Menominee",
+  "America/Merida": "Central Time - Merida",
+  "America/Mexico_City": "Central Time - Mexico City",
+  "America/Monterrey": "Central Time - Monterrey",
+  "America/Rankin_Inlet": "Central Time - Rankin Inlet",
+  "America/Indiana/Tell_City": "Central Time - Tell City, Indiana",
+  "America/Winnipeg": "Central Time - Winnipeg",
+  "America/Cancun": "Eastern Standard Time - Cancun",
+  "America/Jamaica": "Eastern Standard Time - Jamaica",
+  "America/Panama": "Eastern Standard Time - Panama",
+  "America/Guayaquil": "Eastern Standard Time - Guayaquil",
+  "America/Lima": "Peru Time - Lima",
+  "America/Manaus": "Amazon Time - Manaus",
+  "America/Barbados": "Atlantic Standard Time - Barbados",
+  "America/Martinique": "Atlantic Standard Time - Martinique",
+  "America/Puerto_Rico": "Atlantic Standard Time - Puerto Rico",
+  "America/Santo_Domingo": "Atlantic Standard Time - Santo Domingo",
+  "America/Detroit": "Eastern Time - Detroit",
+  "America/Grand_Turk": "Eastern Time - Grand Turk",
+  "America/Indiana/Indianapolis": "Eastern Time - Indianapolis",
+  "America/Iqaluit": "Eastern Time - Iqaluit",
+  "America/Kentucky/Louisville": "Eastern Time - Louisville",
+  "America/New_York": "Eastern Time - New York",
+  "America/Port-au-Prince": "Eastern Time - Port-au-Prince",
+  "America/Thunder_Bay": "Eastern Time - Thunder Bay",
+  "America/Toronto": "Eastern Time - Toronto",
+  "America/Guyana": "Guyana Time ",
+  "America/Asuncion": "Paraguay Time - Asuncion",
```

```suggestion
  "America/Asuncion": "Paraguay Time - Asunción",
```

Not sure about this one, there are others too that have been standardized out in the clean data column.

---

I have asked Diane about accent marks as well, and will add/remove them as she requires once she gets back to me